### DISCUSSION TITLE
Renew you license help

### DISCUSSION THREAD

**Discussion by mbrow081** [datalust/seq-tickets#2287] [Help]
Hello, I am trying to renew a seq license. The user who started the subscription is no longer with the company and we lost access to the account. I've created a new datalust account and submitted the license information into the renew license form. After submitting I am redirected to a page saying "Sorry, that URL was not found". 

Is there a way to renew the subscription as a new account? Or will I be required to start a new subscription?

Thanks for the help!

**Comment by nblumhardt** [comment]
Hi Michael, thanks for reaching out.

We'll be able to get this sorted out for you; can you please mail `support@datalust.co` and let us know the subscription id? Thanks!
